nn_belyi/AI_functions.py

```diff
@@ -0,0 +1,101 @@
+# Functions related to setting up the network.
+
+# INTERESTING: Try to train a network that recognizes the (0,1)-dessign from a (0, inf)-dessign 
+
+###############################################################################################
+# Utilities for creating and fine-tuning neural networks in Keras.
+
+
+import os, sys, scipy.io, scipy.linalg, time, random, pickle
+import warnings
+warnings.simplefilter(action='ignore', category=FutureWarning)
+warnings.simplefilter(action='ignore', category=RuntimeWarning)
+os.environ['TF_CPP_MIN_LOG_LEVEL']='2' #don't display warnings; only errors
+
+import numpy as np
+import tensorflow as tf
```

Comment:
are we using anything to setup the environment? If not, should we add something to the README about dependencies?

Reply:
I'm not sure about the general plan. If the repo becomes a package, we can have the usual `setup.py` framework to handle dependencies.

Reply:
I guess I was just thinking that this part of the code seems self contained, so you could have a Pipfile (or whatever) in this directory to setup the env as the script gets developed.